docs/develop/01_sdk/02_cookbook/01_dids/06_full_did_tx.md

```diff
@@ -0,0 +1,51 @@
+---
+id: full-did-batch
+title: Build DID Extrinsics
+---
+
+import TsJsBlock from '@site/src/components/TsJsBlock';
+
+import FullDidSignTx from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/did/08_full_did_sign_tx.ts';
+import FullDidBatch from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/did/08_full_did_batch.ts';
+
+A DID can be used to sign extrinsics.
+But not every extrinsics can be sign using a DID.
+The Spiritnet blockchain offers two flavours of extrinsics.
+
+The first flavour can only be called using an account.
+These extrinsic require fee payment and need therefore be send from an account with sufficient balance.
```

Comment:
```suggestion
These extrinsic require fee payment and need therefore be sent from an account with sufficient balance.
```

Reply:
It sounds like the second flavour of extrinsics do not involve any fee payment. I would probably switch them around, mentioning that there are some operations that require a DID signature. Everything else is the same. Both involve fee payment and regular account signatures.

Reply:
But we say in the second flavor that these need to be wrapped using the first flavor inorder for fees to be paid. So i think it's clear that both need to be paid.

Also we don't have terminology for these two types. We should give them names so that we can distinguish them. Account Extrinsic and DID Extrinsic?

Reply:
At least I had this doubt when going over the text, I think at least the text can be reworded to explain the concept a bit differently.
I am ok with the Account vs DID Extrinsic distinction.

Reply:
I tried to make it clearer. [e16cf85](https://github.com/KILTprotocol/docs/pull/183/commits/e16cf85b28773e1eb24a03ee8b74d28ee4a30b4c)